python/ray/actor.py

```diff
@@ -411,7 +411,9 @@ def _remote(self,
                 max_restarts=None,
                 max_task_retries=None,
                 name=None,
-                detached=False):
+                detached=False,
+                placement_group_id=None,
+                bundle_index=None):
         """Create an actor.
```

Comment:
Can you document these args? In particular, does None mean any index is allowed?